waspc/data/Generator/templates/server/src/auth/session.ts

```diff
@@ -0,0 +1,102 @@
+{{={= =}=}}
+import { Request as ExpressRequest } from "express";
+
+import { type {= userEntityUpper =} } from "../entities/index.js"
+import { type SanitizedUser } from '../_types/index.js'
+
+import { auth } from "./lucia.js";
+import {
+  throwInvalidCredentialsError,
+  deserializeAndSanitizeProviderData,
+} from "./utils.js";
+
+import prisma from '../dbClient.js';
+
+// Lucia's Session is tied to the Auth model,
+// so we keep the Auth ID in the session.
+export async function createSession(authId: string) {
+  return auth.createSession(authId, {});
+}
```

Comment:
I prefer not to type wrapper functions since most of the time we don't have the 3rd party lib type available to us and typing the return value here would be just `ReturnType<typeof auth.createSession>` which feels pointless since of course it's that since we are returning the `auth.createSession` call.

Reply:
About the comment, it is a bit redundant in its current form, I'll update it a bit.

Reply:
But here it's relatively easy to import `Session` from `lucia` and type it with `Promise<Session>`, so we can do it 👍

Reply:
Ok! I will let you resolve this comment, just in case you wanted to do this but forgot (because I see no :rocket: ). I would vote for typing it, but am ok with skipping.